### PR TITLE
chore: Updates to clang-format 11.0.0

### PR DESCRIPTION
**Pull request by purew**
To update current installations, do

```bash
rm -rf mason
make fmt
```

### CODE REVIEW

File: scripts/bash_utils.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ function setup_mason {
   if [ ! -f mason/mason ] ; then
       echo "Installing mason"
       mkdir -p ./mason
-      curl -sSfL https://github.com/mapbox/mason/archive/88f931cf8a327e3ab4802272e47a4d192f846fda.tar.gz \
+      curl -sSfL https://github.com/mapbox/mason/archive/6e12456fd85e842eda63e87e5b706a7e961e522d.tar.gz \
```
Comment (purew) [inline scripts/bash_utils.sh:19]: A newer `mason` release which contains clang-format 11.

File: src/bindings/python/python.cc [hunk 2]
```diff
@@ -43,26 +43,32 @@ namespace py = pybind11;
...
-      .def("route", [](vt::actor_t& self, std::string& req) { return self.route(req); },
-           "Calculates a route.")
-      .def("locate", [](vt::actor_t& self, std::string& req) { return self.locate(req); },
-           "Provides information about nodes and edges.")
-      .def("optimized_route",
-           [](vt::actor_t& self, std::string& req) { return self.optimized_route(req); },
-           "Optimizes the order of a set of waypoints by time.")
+      .def(
+          "route", [](vt::actor_t& self, std::string& req) { return self.route(req); },
+          "Calculates a route.")
```
Comment (kevinkreiser) [inline src/bindings/python/python.cc:48]: this is less good but i see why they did it for left side alignment

File: valhalla/midgard/tiles.h [hunk 3]
```diff
@@ -301,7 +301,8 @@ template <class coord_t> class Tiles {
    */
   int32_t LeftNeighbor(const int32_t tileid) const {
     return tileid - ((tileid / ncolumns_) * ncolumns_) > 0 ? tileid - 1
-                                                           : wrapx_ ? tileid + ncolumns_ - 1 : tileid;
+           : wrapx_                                        ? tileid + ncolumns_ - 1
+                                                           : tileid;
```
Comment (kevinkreiser) [inline valhalla/midgard/tiles.h:305]: fail :smile:

**Review by kevinkreiser — DISMISSED** [review]

**Comment by mandeepsandhu** [comment]
Is the new version wrapping at 80 lines now? saw a few places where it seemed to have wrapped at 80 chars.

**Comment by purew** [comment]
> Is the new version wrapping at 80 lines now? saw a few places where it seemed to have wrapped at 80 chars.

No idea. I would expect a much bigger diff if the line wrap threshold was decreased.

**Review by kevinkreiser — APPROVED** [review]

**Review by nilsnolde — APPROVED** [review]

**Comment by nilsnolde** [comment]
so @all, purge your current clang-format executable or CI will fail

**Pull request merged**